setup.py

```diff
@@ -44,10 +44,10 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.27",
-        "qiskit ~= 1.0",
+        "pytket ~= 1.28",
```

Comment:
Can we go with the versions in #354 which are more recent?

Reply:
Yes we can